tools/pontoon/search_filters.md

```diff
@@ -0,0 +1,30 @@
+# Search and filters
+
+A string in Pontoon can be in one of these states:
+* *Missing*: string is not available in the localized file.
```

Comment:
Uhm, that's not true, a fuzzy string **is** in the file, there's no way to mark a string as fuzzy in Pontoon, right?

Using for now: `string is not available in the localized file and doesn’t have any translations or suggestions in Pontoon`